gameboy_opengl/src/lib.rs

```diff
@@ -4,7 +4,7 @@ mod screen;
 use crate::native_rtc::NativeRTC;
```

Comment:
This file needs to be updated to use the new controller functions in the `Gameboy` struct. The way it is now, the controls don't work on the native version

Reply:
It looks like the controls are now plugged back into the gameboy struct. There is an existing bug with the up and down arrow buttons not working that I should fix.

Reply:
The controls seem to work fine for me. What game are you testing with?

Reply:
I am testing with a pokemon rom. The rom might be corrupted

Reply:
Tested it with a more official ROM and it is working fine